syft.hcl

```diff
@@ -0,0 +1,16 @@
+description = "A CLI tool and go library for generating a Software Bill of Materials (SBOM) from container images and filesystems."
+binaries = ["syft"]
+
+darwin {
+  source = "https://github.com/anchore/syft/releases/download/v${version}/syft_${version}_darwin_amd64.zip"
+}
+
+linux {
+  source = "https://github.com/anchore/syft/releases/download/v${version}/syft_${version}_linux_amd64.tar.gz"
```

Comment:
These sources can be combined into one at the top level if you prefer, by using `${os}` to replace linux and darwin.